include/fixed_containers/enum_array.hpp

```diff
@@ -172,3 +172,14 @@ class EnumArray
     constexpr ValueArrayType& values() { return IMPLEMENTATION_DETAIL_DO_NOT_USE_values_; }
 };
 }  // namespace fixed_containers
+
+// Specializations
+namespace std
+{
+template <typename L, typename T>
+struct tuple_size<fixed_containers::EnumArray<L, T>> : std::integral_constant<std::size_t, 0>
```

Comment:
So this prevents people from doing `auto [foo,bar,blah] = EnumArray<T,N>` and expecting it to work?

Reply:
Yeah, this will cause a compilation failure instead.